src/components/Card/Card.scss

```diff
@@ -17,18 +17,19 @@
 }
 
 .Header {
-  padding: spacing() spacing() 0;
+  padding: spacing() spacing() rem(12px) spacing();
```

Comment:
This is not the first time I've seen the need for `12px` and I wonder if we need to add base * 3 in our spacing Sass map?

@kaelig Is this something you'd have an opinion about?

Reply:
I dislike our spacing API primarily because it actively works against a shared vocabulary between designers and devs. Design tools don’t allow named spacing values, but our spacing API enforces it.

(I should also point out that other values fit the system—24px is a valid spacing value, same with 28, and 36, and 40…)

Reply:
Resolving this. Opened an issue for discussion [here](https://github.com/Shopify/polaris-react/issues/1007).